models/aeif_cond_beta_multisynapse.h

```diff
@@ -454,9 +500,38 @@ aeif_cond_beta_multisynapse::set_status( const DictionaryDatum& d )
   // consistent.
   Archiving_Node::set_status( d );
 
+  /**
+   * Here is where we must update the recordablesMap_ if new receptors
+   * are added!
+   */
```

Comment:
Should be a normal comment, not a Doxygen comment.

Reply:
Done in eb41277